questdb-rs/Cargo.toml

```diff
@@ -41,6 +42,14 @@ mio = { version = "0.8.8", features = ["os-poll", "net"] }
 chrono = "0.4.30"
 
 [features]
+default = ["tls-webpki-certs"]
```

Comment:
shouldn't we include also the `tls-native-certs` feature by default?

Reply:
At the moment it's not really required for existing users.
Is it really worth adding the extra dependency for them?

Reply:
well, if the default behaviour is to combine both bundled and system trust-store (as discussed earlier) then it's needed anyway, isn't it?

Reply:
There's no default behaviour on the Rust side. The default behaviour is going to be on the Python side.
The FFI includes both.

Reply:
by thinking: if we consider this to be a good default why not make it the default for everyone? 
and if you care about a slim dependencies then you can always exclude default features. wdyt?

Reply:
ok, fair. I am not that familiar with the Rust culture. so I chose to trust your instinct for the expected behaviour.